server/src/services/Teams.ts

```diff
@@ -65,14 +69,49 @@ export class Teams implements IService {
         }).then();
     }
 
+    @Action
+    @Description("A cool way to post an issue to Teams using adaptive cards")
+    postIssue(): void
+    {
+        if (!this._outgoing) return;
+
+        fetch(this._outgoing, {
+            method: 'POST',
+            body: JSON.stringify(AreaCards.issueFormat(this._authorName, this._authorImage, this._repository, this._repositoryLink, this._issueId, this._issue, this._issueLink, this._message, this._datetime)),
+            headers: {'Content-Type': 'application/json'} 
+        }).then();
+    }
+
+    // @area.Action
+    // postTask(): void
+    // {
+    //     if (!this._outgoing) return;
+
+    //     fetch(this._outgoing, {
+    //         method: 'POST',
+    //         body: JSON.stringify(
+    //             {}
+    //         ),
+    //         headers: {'Content-Type': 'application/json'} 
+    //     })
+    // }
+
     _hour: number | undefined;
     _minute: number | undefined;
     _day: number | undefined;
     _month: number | undefined;
     _year: number | undefined;
     _authorId: ustring;
     _authorName: ustring;
+    _authorImage: ustring;
+    _title: ustring;
+    _datetime: Date | undefined = new Date(Date.now());
     _message: ustring;
     _subject: ustring;
+    _repository: ustring;
+    _repositoryLink: ustring;
+    _issue: ustring;
+    _issueId: number | undefined;
+    _issueLink: ustring;
     _outgoing: nstring;
```

Comment:
We really should use interfaces for Actions and Reaction data imo

Reply:
check out the weather pr